tests/QuikGraph.Tests/Algorithms/Contracts/Feeder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuikGraph.Algorithms;
+
+namespace QuikGraph.Tests.Algorithms.Contracts
+{
+    public static class Feeder
```

Comment:
I'm not sure if this is a good name for this class. but couldn't come up with anything else. Do you have any suggestions?

Reply:
I preparing a commit to fix those kind of stuff ;-)